src/Traits/SecurityTrait.php

```diff
@@ -10,6 +10,8 @@ trait SecurityTrait
     /**
      * Security action url.
      *
+     * @Groups({"create", "update"})
+     *
```

Comment:
we are not validating any of these properties, there seems to be a missing consistency between usage of @Assert validation between traits.